docs/en/metadata_oidc_rp.rst

```diff
@@ -59,6 +59,12 @@ The RP Metadata of type **"openid_relying_party"** MUST contain at least the fol
   * - **id_token_signed_response_alg**
     - See `OpenID.Registration#ClientMetadata`_.
     - |spid-icon| |cieid-icon| 
+  * - **id_token_encrypted_response_alg**
+    - See `OpenID.Registration#ClientMetadata`_. Optional for CIE id, required for SPID. If this is requested, the response MUST signed then encrypted, with the result being a Nested JWT, as defined in :rfc:`OpenID.Core`_ Section 10.
```

Comment:
the **id_token** MUST be signed then encrypted

Reply:
https://github.com/italia/spid-cie-oidc-docs/pull/55/commits/0e038ebd85305c7f69305fc305dded8379de4022